### DISCUSSION TITLE
Quick Check 2023

### DISCUSSION THREAD

**Discussion by davidstanke** [dora-team/dora.dev#523] [Announcements]
The new quick check is nearly ready. Preview it at https://dora.dev/quickcheck/2023/ and share your feedback here.
[FAQ for users](https://dora.dev/faq/quickcheck_2023/#whats-new-in-the-2023-quick-check)

([notes on the implementation](https://github.com/dora-team/dora.dev/tree/main/svelte/quick-check-2023))

**Comment by Tiberriver256** [comment]
"deploy code to production or release it to end users"

With feature flags becoming commonplace, the difference in time between deployed to production and released to end users can be significant.

I suggest that be more specific on one or the other, not both.

Looking great btw 👍

**Reply by davidstanke** [reply]
Great point! We pull the question text verbatim from DORA's State of DevOps survey questions, and those questions—so far—haven't accommodated that time span between deploy and release. It's certainly worth considering for future research!

**Comment by louisbennett** [comment]
<img width="1089" alt="image" src="https://github.com/dora-team/dora.dev/assets/1386462/f34dc6ef-a7a6-4539-b7e2-3fd11347327f">
For this question, it's quite hard to distinguish what value one is entering – there is not a minimum or maximum shown, nor incremental feedback on values. (I believe it's meant to span from 0–100%, but missing labels.)
Browser: Safari 17.3 (19617.2.4.11.8)

**Reply by davidstanke** [reply]
Thanks for raising this, @louisbennett! It turns out that this was a Safari-specific bug (props for specifying that in your comment!), which has [now been fixed](https://github.com/dora-team/dora.dev/pull/526).

**Comment by steve-fenton-octopus** [comment]
How would you feel about dedicating a line break to each of these?

![Key for the quick check results](https://github.com/dora-team/dora.dev/assets/99181436/05ad26a9-33dc-4a25-a133-26b9d01a9b71)

![Key arranged a bit](https://github.com/dora-team/dora.dev/assets/99181436/4f644c11-1cab-4b04-b358-386a20e465f8)

And a bit of an ameteur question... does "standard deviation" here refer to "1 standard deviation"?

**Reply by davidstanke** [reply]
Good ideas! I've opened issues #531 and #530 to track these.

**Comment by daniel-lohausen-consilica** [comment]
I like it, much leaner than the old version. And its easier to compare the industries.

One suggestion: Can you add a link to the DORA-Core Model on the final screen: https://dora.dev/research/  I think this is a vital piece of DORA. Happy to support with a sentene of explanation if that is needed.

**Reply by davidstanke** [reply]
Good idea! I've opened issue #532 to track this.

**Comment by davidstanke** [comment]
Also see #534
